website/app.js

```diff
@@ -123,6 +123,8 @@ export async function main() {
   const sha1ShortList = sha1List.map(sha1 => sha1.substring(0, 6));
   const prNumberList = travisData.map(d => d.pull_request_number);
 
+  console.log(threadCountColumns);
```

Comment:
Is this required?  Seems like debugging code left behind.

Reply:
Nops, it isn't. I've removed that